recruiter/tests.py

```diff
@@ -1,3 +1,46 @@
-# from django.test import TestCase
+import pytest
+from django.contrib.auth.models import User
+from django.db.models import QuerySet
 
-# Create your tests here.
+from recruiter.models import Company, Recruiter
+
+
+@pytest.fixture
+def create_company():
+    company = Company(name="test_company", description="test_company_description", website_url="test_company_web")
+    company.save()
+    return company
+
+
+@pytest.fixture()
+def example_user():
+    user = User.objects.create_user("test", "testPassword")
+    user.save()
+    return user
+
+
+@pytest.fixture
+def create_recruiter(create_company, example_user):
+    recruiter = Recruiter(user=example_user, name="test_recruiter", company=create_company, email="test@email",
+                          phone_number="test_phone")
+    recruiter.save()
+    return recruiter
+
+
+@pytest.mark.django_db
+def test_get_all_companies_returns_all_companies_as_a_query_set(create_company):
```

Comment:
you are not using the fixture in the test body - if you just always need the test data in the DB - perhaps create it in a migration file?

Reply:
I don't think I understand what you mean
I do use the fixture in the test body and I do use both fixtures in the other function as well

Reply:
ok, I see it now, but still, why not create the test data in a migration file? having some data there will make developing the screens easier later